membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -65,6 +65,9 @@ class AttributeController(attributesFromZuora: AttributesFromZuora, commonAction
                 attrs.PaperSubscriptionExpiryDate.foreach {date =>
                   logInfoWithCustomFields(s"$identityId is a paper subscriber expiring $date", customFields("paper-subscriber"))
                 }
+                attrs.PaperSubscriptionExpiryDate.foreach {date =>
```

Comment:
Should this be attrs.GuardianWeeklySubscriptionExpiryDate?

Reply:
good spot - fixed - thanks